taichi/backends/cpu/aot_module_loader_impl.cpp

```diff
@@ -1,56 +1,29 @@
 #include "taichi/backends/cpu/aot_module_loader_impl.h"
+#include "taichi/llvm/llvm_aot_module_loader.h"
 
 #include "taichi/llvm/llvm_offline_cache.h"
 #include "taichi/llvm/llvm_program.h"
 #include "taichi/codegen/codegen_llvm.h"
 
 namespace taichi {
 namespace lang {
-namespace cpu {
 namespace {
 
-class KernelImpl : public aot::Kernel {
- public:
-  explicit KernelImpl(FunctionType fn) : fn_(fn) {
-  }
-
-  void launch(RuntimeContext *ctx) override {
-    fn_(*ctx);
-  }
-
- private:
-  FunctionType fn_;
-};
-
-class AotModuleImpl : public aot::Module {
+class AotModuleImpl : public LLVMAotModule {
```

Comment:
Nit: Let's follow the style guide https://google.github.io/styleguide/cppguide.html#General_Naming_Rules and name this `LlvmAotModule`

Reply:
Thanks, let me rename that in the following PR

Reply:
Done, thanks!